### ISSUE TITLE
Tell me ONCE about anchor loss is enough.

### ISSUE THREAD

**Issue by be5invis** [fontforge/fontforge#2255]
When pasting glyphs between fonts with anchor lost, this notification is annoying...
![image](https://cloud.githubusercontent.com/assets/240091/7476552/ca8659f0-f380-11e4-9fb1-a28c4350e093.png)

**Comment by davelab6** [comment]
Agree with @andreaslarsen that a `don't notify me again` option would be good, on a session and forever basis.

**Comment by be5invis** [comment]
@davelab6 @andreaslarsen There are many other dialog boxes are still annoying, including the dialog box poped out when auto-instructing a mixed glyph.

**Comment by andreaslarsen** [comment]
Yep, thought about this yesterday too but forgot to post it. I'll update the title to reflect this.

**Comment by davelab6** [comment]
Cool! Please add screenshots of everything you found annoying. 

@andreaslarsen I think we should use the passive Warnings log window instead of all these alerts.

**Comment by andreaslarsen** [comment]
good idea, that way all alerts is in one place and the don't notify me again isn't really needed

**Comment by andreaslarsen** [comment]
@davelab6 why not just go all in and just use the log in the first place?

**Comment by davelab6** [comment]
I'm also happy with that :)
